ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml

```diff
@@ -748,71 +757,50 @@ periodics:
-          - as: e2e-builds
-            commands: TEST_SUITE=openshift/build run-tests
-            openshift_installer:
-              cluster_profile: "aws" # work around lack of metal support
-          - as: e2e-disruptive
-            commands: setup_ssh_bastion; TEST_SUITE=openshift/disruptive run-tests
-            openshift_installer:
-              cluster_profile: "aws" # work around lack of metal support
-          - as: e2e-disruptive-all
-            commands: setup_ssh_bastion; TEST_SUITE=openshift/disruptive run-tests; TEST_SUITE=openshift/conformance run-tests
-            openshift_installer:
-              cluster_profile: "aws" # work around lack of metal support
-          - as: e2e-all
-            commands: TEST_SUITE=openshift/conformance run-tests
-            openshift_installer:
-              cluster_profile: "aws" # work around lack of metal support
-          - as: e2e-image-ecosystem
-            commands: TEST_SUITE=openshift/image-ecosystem run-tests
-            openshift_installer:
-              cluster_profile: "aws" # work around lack of metal support
-          - as: e2e-image-registry
-            commands: TEST_SUITE=openshift/image-registry run-tests
-            openshift_installer:
-              cluster_profile: "aws" # work around lack of metal support
-          - as: e2e-network-stress
-            commands: TEST_SUITE=openshift/network/stress run-tests
-            openshift_installer:
-              cluster_profile: "aws" # work around lack of metal support
           - as: e2e-serial
-            commands: TEST_SUITE=openshift/conformance/serial run-tests
-            openshift_installer:
-              cluster_profile: "aws" # work around lack of metal support
-          - as: e2e-upgrade
-            commands: TEST_SUITE=all run-upgrade-tests
-            openshift_installer:
-              cluster_profile: "aws" # work around lack of metal support
-              upgrade: true
-          - as: e2e-upgrade-all
-            commands: TEST_SUITE=all run-upgrade-tests; TEST_SUITE=openshift/conformance run-tests
-            openshift_installer:
-              cluster_profile: "aws" # work around lack of metal support
-              upgrade: true
-          - as: e2e-upgrade-partial
-            commands: TEST_SUITE=all TEST_UPGRADE_OPTIONS=abort-at=random run-upgrade-tests; TEST_SUITE=openshift/conformance/parallel run-tests
-            openshift_installer:
-              cluster_profile: "aws" # work around lack of metal support
-              upgrade: true
-          - as: e2e-upgrade-rollback
-            commands: TEST_SUITE=all TEST_UPGRADE_OPTIONS=abort-at=100 run-upgrade-tests; TEST_SUITE=openshift/conformance/parallel run-tests
-            openshift_installer:
-              cluster_profile: "aws" # work around lack of metal support
-              upgrade: true
+            steps:
+              cluster_profile: packet
+              workflow: baremetalds-e2e-serial-ipv4
```

Comment:
Why serial?

Reply:
Nearly all platforms support e2e-serial in cluster-bot, it is added to preserve the consistency at least partially.